src/object/srv_obj_migrate.c

```diff
@@ -476,6 +476,9 @@ migrate_pool_tls_lookup_create(struct ds_pool *pool, int version,
 
 	tls = migrate_pool_tls_lookup(pool->sp_uuid, version);
 	D_ASSERT(tls != NULL);
+	if (opc == RB_OP_REINT)
+		pool->sp_reintegrating++;
```

Comment:
Is multiple parallel reintegrations to same pool allowed? If not, I suggest use 'true' or 'false' and add some proper assertions.

Reply:
Since our rebuild/reintegration is version based, I do not think there will be concurrent multiple reintegrations.